app/services/submit_partner_service.rb

```diff
@@ -44,10 +44,15 @@ def regular_transactions
   end
 
   def state_benefits
-    form = PartnerBenefitsForm.from_session(@cfe_session_data)
-    return [] if form.benefits.blank?
+    benefits_form = PartnerBenefitsForm.from_session(@cfe_session_data)
+    housing_benefit_form = PartnerHousingBenefitForm.from_session(@cfe_session_data)
+    return [] if benefits_form.benefits.blank? && !housing_benefit_form.housing_benefit
 
-    CfeParamBuilders::StateBenefits.call(form)
```

Comment:
This change suggests that the StateBenefits param builder has an optional second parameter that maybe isn't optional any more?